### PR TITLE
[Documentation][Book][Orders] Fix

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | yes |
| New docs?    | no|
| Related tickets | |
| License         | MIT |

### CODE REVIEW

**Comment by bendavies** [comment]
hmm that reads badly now in english.  
you want:
`with a state machine transition` or
`with state machine transitions`

File: docs/book/orders/orders.rst [hunk 1]
```diff
@@ -195,8 +195,8 @@ Remember to process the order using order processor and then flush the order man
 
     $order->addPayment($payment);
 
-Shipping a Shipment with a state machine transition
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Completing a payment with state machine transition
```
Comment (lchrusciel) [inline docs/book/orders/orders.rst]: Why `Shipping a Shipment` has two letters capitalized and `Completing a payment` has only one?

**Comment by TheMadeleine** [comment]
@bendavies Good catch. `with a state machine transition` is then one I want.

**Comment by pjedrzejewski** [comment]
👍

**Pull request merged**